core/templates/pages/exploration-editor-page/settings-tab/settings-tab.component.html

```diff
@@ -133,7 +133,7 @@ <h3 class="e2e-test-settings-container" tabindex="0">Basic Settings</h3>
               </div>
             </div>
             <div class="text-right">
-              <button type="button" class="btn btn-secondary e2e-test-open-preview-summary-modal"
+              <button type="button" class="btn btn-secondary e2e-test-open-preview-summary-modal click-to-see-preview-summary"
```

Comment:
The new class name does not follow conventions. If you want to add a name just for e2e tests then the class must start with e2e-test. There is already such a class here so you might as well reuse that one.

Reply:
updated ✅